README.md

```diff
@@ -26,6 +26,9 @@ We are iterating and looking for feedback and collaboration, so please [**let us
  - Predictions
  - Datastore
  - Hub Events (Listening to the Amplify events)
+ - iOS Error Events in Amplify Analytics
+
+> Amplify for Flutter is in preview, which should be considered an alpha version and not recommended for production deployments. We are iterating and looking for feedback and collaboration, so please [**LET US KNOW YOUR FEEDBACK**](https://github.com/aws-amplify/amplify-flutter/issues/5) on our direction and roadmap.
```

Comment:
Is this going to be rebased on top of https://github.com/aws-amplify/amplify-flutter/pull/38/files? It appears to contain duplicate changes.

Reply:
Hi @jamesonwilliams this is a new PR.  By making our github repo public, we had to recreate all of our original PRs and delete the old ones (I'll delete pull 38 that you reference right now).

Reply:
I've looked at and implemented all your comments from the previous, closed PR